workflows/example_multifamily_TMY.osw

```diff
@@ -452,6 +452,16 @@
       },
       "measure_dir_name": "ResidentialPhotovoltaics"
     },
+    {
```

Comment:
Did you update these example workflows manually? If so, they shouldn't be. They are generated from a rake task. However in general, I feel like this demand response measure wouldn't be run by default in any example workflows. Instead, it would be stored in the top-level measures directory and added after BuildExistingModel (sort of like how ResidentialSimulationControls works).